spec.bs

```diff
@@ -2140,14 +2220,28 @@ To <dfn>fetch script</dfn> given a [=URL=] |url|:
     Issue: Stop using "`no-cors`" mode where possible
     (<a href="https://github.com/WICG/turtledove/issues/667">WICG/turtledove#667</a>).
   1. Let |script| be null.
-  1. [=Fetch=] |request| with [=fetch/useParallelQueue=] set to true, and
-    [=fetch/processResponseConsumeBody=] set to the following steps given a [=response=] |response|
-    and null, failure, or a [=byte sequence=] |responseBody|:
-    1. If [=validate fetching response=] with |response|, |responseBody| and "`text/javascript`"
-      returns false, set |script| to failure and return.
-    1. Set |script| to |responseBody|.
+  1. Let |allowCrossOriginTrustedScoringSignalsFrom| be an empty [=list=].
+  1. Let |fetchController| be the result of [=fetching=] |request| with [=fetch/useParallelQueue=]
+    set to true, and [=fetch/processResponse=] set to the following steps given
+    a [=response=] |response|:
+    1. If the result of [=validating fetching response headers=] given |response| is false:
+      1. [=fetch controller/Abort=] |fetchController|.
+      1. Set |script| to failure.
+      1. Return.

```

Comment:
Well, we don't want to parse headers and try to read the body if it's a failure? It might coincidentally work but I don't want to think about what exactly it will do?